### ISSUE TITLE
Playground link present in Readme is broken

### ISSUE THREAD

**Issue by Wajeed-msft** [OfficeDev/microsoft-teams-ui-component-library#149]
### Reproduction

[Playground](https://dev.teams.microsoft.com/storybook/main/index.html) link present in  [Readme.md](https://github.com/OfficeDev/microsoft-teams-ui-component-library/blob/main/README.md) file is broken. 

![image](https://user-images.githubusercontent.com/31851992/198946126-10a8b967-47a4-4965-9fc3-834d31d95a78.png)

**Error:**
![image](https://user-images.githubusercontent.com/31851992/198946077-0ecd2ff7-352a-480b-aade-9df5609db910.png)

Could you please help to fix this? 

Note: This is also linked from here [Microsoft Teams UI Kit](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/design/design-teams-app-overview#microsoft-teams-ui-kit).

**Comment by thure** [comment]
Hi @Wajeed-msft — I'm the former developer of this library. There is a version of this library's Storybook hosted on Chromatic: https://63587347138fdad13ed63ccd-omfbjvvebn.chromatic.com/?path=/story/ui-templates-dashboards--default

As far as I know this library is essentially deprecated, but, since Microsoft’s layoffs impacted the teams working on this, the developers now responsible for this repo might not have gotten around to announcing the deprecation.

Please consider developing using [Fluent Blocks](https://github.com/OfficeDev/fluent-blocks) instead of this library. The team developing the Teams Toolkit has also committed to using Fluent Blocks, last I heard, so the resource you linked is likely to change soon.

**Comment by pglombardo** [comment]
That's unfortunate but thanks for the update @thure.  Since some of us are spending weeks prototyping against these libraries, Microsoft should give some hard guidance on what to use...

**Comment by thure** [comment]
Yes, they should, though MS did not sufficiently fund these projects during my time at MS and the community using these projects were not very interested in contributing, so naturally the projects withered. 

Speaking as someone who was very familiar with MS’s offering in this space as of Sept 2022, @pglombardo I believe your best bet is to at least use [Fluent UI v9](https://react.fluentui.dev/). I can’t guarantee [Fluent Blocks](https://github.com/OfficeDev/fluent-blocks) will continue to be developed, but you could fork the project and make changes as needed if it suits your project.

**Comment by thure** [comment]
Speaking as someone who was very familiar with MS’s offering in this space as of Sept 2022, @pglombardo I believe your best bet is to at least use [Fluent UI v9](https://react.fluentui.dev/). I can’t guarantee [Fluent Blocks](https://github.com/OfficeDev/fluent-blocks) will continue to be developed, but you could fork the project and make changes as needed if it suits your project. I may still be able to help merge pull requests.

Some community interest in Fluent Blocks could compel Microsoft to pick it back up.

**Comment by msm-msm20** [comment]
Hello 
Is there any way to show code from Storybook to know how to use React-teams components?
